src/App.tsx

```diff
@@ -113,6 +119,7 @@ export const App: React.FC = () => {
               deleteTodo={deleteTodo}
               isCompleted={isCompleted}
               visibleLoader={visibleLoader}
+              setVisibleLoader={setVisibleLoader}
             />
 
             <footer className="todoapp__footer" data-cy="Footer">
```

Comment:
It seems to me that footer is also better to move it to a separate component

Reply:
it is checklist hint for your comment with naming of components